src/puppetlabs/kitchensink/core.clj

```diff
@@ -150,10 +150,9 @@ to be a zipper."
 (defn lines
   "Returns a sequence of lines from the given filename"
   [filename]
-  (-> filename
-    (fs/file)
-    (reader)
-    (line-seq)))
+  (with-open [file-reader (reader (fs/file filename))]
+    ;; line seq is lazy and file-reader gets closed
```

Comment:
I'm not sure I consider this a breaking change since there's nothing in the description about the lines being returned being lazy.  I think it's good to ensure that the reader is explicitly closed.  Unless we were to change the implementation to return the reader back to the caller so that it could close it after consuming the lines, which would be a breaking change, not sure if there's a way to fix the bug and preserve the lazy behavior.